kucher/view/main_window/telega_control_widget/task_specific_status_widget/fault_status_widget.py

```diff
@@ -22,6 +22,7 @@
 
 from .base import StatusWidgetBase
 
+import yaml
```

Comment:
Please move the third-party import above the local imports, as required by PEP8 (standard first, then third-party, then local). In this case, it should be either above or under the PyQt5 import group.

Also, is PyYAML does not seem to be listed as a dependency.